org.ekstep.questionunit.sequence-1.1/renderer/js/seq-template.js

```diff
@@ -1,4 +1,5 @@
 var SEQController = SEQController || {};
+SEQController.attempted = false;
```

Comment:
Dont define in every plugin. Define in questionUnit plugin & call setter & getter method to get the value of attempted.
org.ekstep.questionUnit
{
  attempted: false,
  get attempted() {
    return this.attempted;
  }
  set attempted(value) {
    this.attempted = value;
  }
}